sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -318,6 +320,7 @@ export class EventHubClient {
    * @throws {AbortError} Thrown if the operation is cancelled via the abortSignal.
    */
   async getProperties(abortSignal?: AbortSignalLike): Promise<EventHubProperties> {
+    throwErrorIfConnectionClosed(this._context);
```

Comment:
Should we add a @throws section to the JSDOC for this error? Maybe add something like "Create a new EventHubsClient and call getProperties when encountering this error"